backend/src/main/resources/ValidationMessages.properties

```diff
@@ -0,0 +1,10 @@
+# IMPORTANT: This file needs to be at src/main/resources/ValidationMessages.properties and named this way for spring to pick it up!!
+
```

Comment:
[MED–TICK]: create a ticket for this issue, i reckon the message template is not in its final form, replase `attribute.*.*` with `validation.*.*`